src/main/java/ch/hearc/cafheg/infrastructure/persistance/AllocataireMapper.java

```diff
@@ -13,65 +15,68 @@
 
 public class AllocataireMapper extends Mapper {
 
+    private static final Logger log = LoggerFactory.getLogger(AllocataireMapper.class);
+
     private static final String QUERY_FIND_ALL = "SELECT NOM,PRENOM,NO_AVS FROM ALLOCATAIRES";
     private static final String QUERY_FIND_WHERE_NOM_LIKE = "SELECT NOM,PRENOM,NO_AVS FROM ALLOCATAIRES WHERE NOM LIKE ?";
     private static final String QUERY_FIND_WHERE_NUMERO = "SELECT NO_AVS, NOM, PRENOM FROM ALLOCATAIRES WHERE NUMERO=?";
 
     public List<Allocataire> findAll(String likeNom) {
-        System.out.println("findAll() " + likeNom);
+        log.info("findAll() called with likeNom='{}'", likeNom);
         Connection connection = activeJDBCConnection();
         try {
             PreparedStatement preparedStatement;
             if (likeNom == null) {
-                System.out.println("SQL: " + QUERY_FIND_ALL);
+                log.debug("Executing SQL: {}", QUERY_FIND_ALL);
```

Comment:
This should be in the trace level in my opinion. It's a detailed information.
```suggestion
                log.trace("Executing SQL: {}", QUERY_FIND_ALL);
```